brands/tourism/hotel.json

```
@@ -510,6 +589,15 @@
       "tourism": "hotel"
     }
   },
+  "tourism/hotel|Moxy": {
+    "tags": {
+      "brand": "Moxy",
+      "brand:wikidata": "Q1053170",
```

This is the QID for Courtyard. I’ve created [Q70287020](https://www.wikidata.org/wiki/Q70287020) for Moxy:

```suggestion
      "brand:wikidata": "Q70287020",
```

Also delete the `brand:wikipedia` tag, since there’s no corresponding Wikipedia article yet.

---

I updated the wikidata but left the wikipedia since that article lists moxy and we do that for other marriott brands.

---

It’s problematic that `brand:wikipedia` and `brand:wikidata` are now inconsistent with each other. Data consumers expect the two tags to be more or less equivalent. (If I’m not mistaken, there are bots going around updating `wikipedia` tags to match `wikidata` tags with the understanding that `wikipedia` is less stable than `wikidata`.)

If the “Marriott International” article had a whole section titled “Moxy”, then that might make a better case for `brand:wikipedia=en:Marriott International#Moxy`. But currently all the article has is a single redlink – no better than what a “List of hotels” article would have.